client/src/components/common/boost/BoostCalibration.tsx

```diff
@@ -0,0 +1,86 @@
+import React , {useCallback, useState} from 'react';
+import {Button, Card, InputGroup, FormControl} from 'react-bootstrap';
+import {setCalibration as sendCalibrationValue, resetCalibration} from 'api/common/powerModel';
+// import {useSensorData} from 'api/v2/sensors/data';
+
+/**
+ * Boost distance claibration input form
+ * 
+ * @returns {React.Component} Component
+ */
+function BoostCalibrationInput() {
+    const [calibrationValue, setCalib] = useState('');
+
+    const handleCalibrationChange = useCallback(
+        (event) => {
+        setCalib(event.target.value);
+        },
+        [setCalib],
+    );
+
+    const onSet = useCallback(
+        () => {
+          const calibValue: number = parseInt(calibrationValue, 10);
+          if (Number.isNaN(calibValue)) {
+            alert("Please enter an integer value only!");
+          }
+          else {
+            console.log(calibValue);
+            sendCalibrationValue(parseInt(calibrationValue, 10));
+          }
+          setCalib('');
```

Comment:
Only reset the form if it passed validation!

Reply:
I thought it would be a good idea to clear it for the user, since the input is wrong anyways

Reply:
I would disagree that this is useful. My thinking is that if a user tries to submit something invalid, they'd want to see what what they typed to see what was wrong with it. If you think about forms with validation that you fill in on the internet, most would not clear your input if it's invalid unless it's something sensitive (like a password).

I think just as a general rule in UI/UX design, doing something unexpected is usually not a good experience for the user, in this case clearing a form when they only wanted to submit.